src/reporter.cc

```diff
@@ -38,6 +38,9 @@ void BenchmarkReporter::PrintBasicContext(std::ostream *out,
 
   Out << LocalDateTimeString() << "\n";
 
+  const SystemInformation &sys_info = context.sys_info;
+  Out << "System Name "<< sys_info.name << "\n";
```

Comment:
Is this printed only for the json dumper, or for all of them?
Put differently, i don't think it should be in the console output, it is getting rather crowded already.